src/object.js

```diff
@@ -10,6 +10,34 @@ Sk.builtin.object = function () {
 };
 
 /**
+ * Special method look up. First try getting the method via
+ * internal dict and getattr. If getattr is not present (builtins)
+ * try if method is defined on the object itself
+ *
+ * Return null if not found or the function
+ */
+Sk.builtin.object._PyObject_LookupSpecial = function(op, str) {
```

Comment:
Might be a good idea it's now showing up on the python side of things. After I merge with master `dir()` reports it. It can be added to the internal list but it'll still be there.